src/interfaces/bitcoin/transactions.ts

```diff
@@ -1,62 +1,63 @@
-  version: number;
-  locktime: number;
-  vin: {
-    txid: string;
-    vout: number;
-    prevout: Vout;
-    scriptsig: string;
-    scriptsig_asm: string;
-    is_coinbase: boolean;
-    sequence: string;
-  }[];
-  vout: Vout[];
-  size: number;
-  weight: number;
-  fee: number;
-  status: TxStatus;
-}
-
-export interface Vout {
-  scriptpubkey: string;
-  scriptpubkey_asm: string;
-  scriptpubkey_type: string;
-  scriptpubkey_address: string;
-  value: number;
-}
-
-export interface TxStatus {
-  confirmed: boolean;
-  block_height: number;
-  block_hash: string;
-  block_time: number;
-}
-
-export interface TxMerkleProof {
-  block_height: number;
-  merkle: string[];
-  pos: number;
-}
-
-export interface TxOutspend {
-  spent: boolean;
-  txid: string;
-  vin: number;
-  status: TxStatus;
-}
-
-export interface TxInstance {
-  getTx: (params: { txid: string }) => Promise<Tx>;
-  getTxStatus: (params: { txid: string }) => Promise<TxStatus>;
-  getTxHex: (params: { txid: string }) => Promise<string>;
-  getTxRaw: (params: { txid: string }) => Promise<string>;
-  getTxMerkleBlockProof: (params: { txid: string }) => Promise<string>;
-  getTxMerkleProof: (params: { txid: string }) => Promise<TxMerkleProof>;
-  getTxOutspend: (params: {
-    txid: string;
-    vout: number;
-  }) => Promise<TxOutspend>;
-  getTxOutspends: (params: { txid: string }) => Promise<Array<TxOutspend>>;
-  postTx: (params: { txhex: string }) => Promise<unknown>;
-}
+export interface Tx {
+  txid: string;
+  version: number;
+  locktime: number;
+  vin: {
+    txid: string;
+    vout: number;
+    prevout: Vout;
+    scriptsig: string;
+    scriptsig_asm: string;
+    is_coinbase: boolean;
+    sequence: string;
+  }[];
+  vout: Vout[];
+  size: number;
+  weight: number;
+  fee: number;
+  status: TxStatus;
+}
+
+export interface Vout {
+  scriptpubkey: string;
+  scriptpubkey_asm: string;
+  scriptpubkey_type: string;
+  scriptpubkey_address: string;
+  value: number;
+}
+
+export interface TxStatus {
+  confirmed: boolean;
+  block_height: number;
+  block_hash: string;
+  block_time: number;
+}
+
+export interface TxMerkleProof {
+  block_height: number;
+  merkle: string[];
+  pos: number;
+}
+
+export interface TxOutspend {
+  spent: boolean;
+  txid: string;
+  vin: number;
+  status: TxStatus;
+}
+
+export interface TxInstance {
+  getTx: (params: { txid: string }) => Promise<Tx>;
+  getTxStatus: (params: { txid: string }) => Promise<TxStatus>;
+  getTxHex: (params: { txid: string }) => Promise<string>;
+  getTxRaw: (params: { txid: string }) => Promise<string>;
+  getTxMerkleBlockProof: (params: { txid: string }) => Promise<string>;
+  getTxMerkleProof: (params: { txid: string }) => Promise<TxMerkleProof>;
+  getTxOutspend: (params: {
+    txid: string;
+    vout: number;
+  }) => Promise<TxOutspend>;
+  getTxOutspends: (params: { txid: string }) => Promise<Array<TxOutspend>>;
+  getTransactionTimes: (params: { txId: string[] }) => Promise<Array<number>>;

```

Comment:
The naming of the method is not consistent with the others. Was it taken from the original repository?

Reply:
`getTransactionTimes` is used by https://github.com/utxostack/rgbpp-explorer , see https://github.com/search?q=org%3Autxostack%20getTransactionTimes&type=code